src/crab/array_domain.hpp

```
@@ -40,7 +40,7 @@ class ArrayDomain final {
   public:
     ArrayDomain() = default;
 
-    explicit ArrayDomain(const BitsetDomain& num_bytes) : num_bytes(num_bytes) {}
+    explicit ArrayDomain(BitsetDomain num_bytes) : num_bytes(std::move(num_bytes)) {}
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: num_bytes

(cppcoreguidelines-pro-type-member-init)